gallery/src/telemetry.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+export function init() {
+  // Filled in via the webpack EnvironmentPlugin
+  /* eslint-disable-next-line no-undef */
+  const apiKey = process.env.PX_API_KEY;
```

Comment:
To verify that this works, you can search for the API key value in the `bundle.js` that was produced by the latest jenkins build, and see that it is present.

Reply:
> To verify that this works, you can search for the API key value in the `bundle.js` that was produced by the latest jenkins build, and see that it is present. Once this code is merged to main, we will be able to see it in action on gallery.sonatype.dev

I was able to verify that the API Key is present in the `bundle.js`